.github/workflows/runner.yml

```
@@ -0,0 +1,34 @@
+name: 🔗 Tiered Pipeline Runner
+on: [push, pull_request]
+
+concurrency:
+  group: ci-${{github.actor}}-${{github.head_ref || github.run_number}}-${{github.ref}}-runner
+  cancel-in-progress: true
+
+jobs:
+  static-checks:
+    uses: ./.github/workflows/static_checks.yml
+
+  android-build:
+    needs: static-checks
+    uses: ./.github/workflows/android_builds.yml
+
+  ios-build:
```

```suggestion
  ios-build:
    name: 🍏 iOS
```

See https://github.com/godotengine/godot/pull/75338#issuecomment-1485488463.